css/styles.css

```diff
@@ -733,6 +735,11 @@ body.documentation .container {
     width: auto;
 }
 
+body.documentation section#documentation .container {
+    margin-left: 10px;
+    margin-right: 20px;
```

Comment:
Do we follow any formatting guideline in this file?
There seem to be properties with just one space on the left and here we have four.

Reply:
http://csscomb.com/ ;-)

Reply:
Totally aware of that 😅  I never dared to unleash an autoformatter on that file.

Just so you know: These are some quick fixes to make the documentation presentable now. I'll present a proposal for a new design soon...